src/jwt_authorizer/tokens.py

```diff
@@ -91,15 +97,23 @@ def parse_ticket(prefix: str, ticket: str) -> Optional[Ticket]:
     ticket_id, secret_b64 = trimmed_ticket.split(".")
     try:
         int(ticket_id, 16)  # Check hex
-        secret = base64.urlsafe_b64decode(add_padding(secret_b64))
+        secret = base64.b64decode(
+            add_padding(secret_b64), altchars=b"-_", validate=True
+        )
+        if secret == b"":
+            raise ValueError("ticket secret is empty")
```

Comment:
This looks slightly different but I'm not sure how it's really different?  Seems legit either way?

Reply:
The `validate=True` is the difference. Without that, Python silently discards invalid base64 characters, which I don't think we want to do here. Unfortunately, there's no `validate` flag to `urlsafe_b64decode` so I have to use the generic function and set `altchars`.

Reply:
Sounds good!